src/c20_client/client.py

```diff
@@ -26,51 +26,71 @@ def do_job(api_key):
         LOGGER.info('Getting job from server...')
         job = requests.get('http://capstone.cs.moravian.edu/get_job')
         job = job.json()
-        LOGGER.info("Job aquired")
+        LOGGER.info("Job has been aquired")
 
     except Exception:
+        LOGGER.error("A connection error has occurred")
         raise NoConnectionError
```

Comment:
Since you are refactoring, pull this out into a function `get_job`.  This will match the level of abstraction for the second function all.